examples/coordShiftTest/xRun.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd ./build 
+make -j 4
+cd ..
+./grid
+
```

Comment:
I think this rather duplicates the functionality of the script in ``bin/bob_make`` so probably shouldn't be commited